### DISCUSSION TITLE
Button link not working?

### DISCUSSION THREAD

**Discussion by ShxDowXD** [maximmax42/Discord-CustomRP#150] [Q&A] [answered]
In the app, I have set up a button that is supposed to take you to my website, but when I click it in discord on my profile nothing happens?

**Comment by maximmax42** [comment, marked as answer]
https://github.com/maximmax42/Discord-CustomRP/wiki/FAQ#when-i-click-on-custom-buttons-in-my-profile-they-dont-work

**Comment by ghost** [comment]
You cannot click your own buttons on Discord, but you can try clicking the buttons with an different account or device.
